scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
generally adding ```set -eu -o pipefail``` will help debug bash scripts for us later when we expand.

Reply:
Doh. Forgot about this.